package.json

```diff
@@ -91,8 +93,10 @@
     "expect-puppeteer": "4.4.0",
     "gettext-parser": "4.0.4",
     "husky": "6.0.0",
+    "jest-environment-jsdom": "27.5.1",
```

Comment:
Is it to force a version in a dependency or something like that?

Reply:
Yes! Without it, jest fails because it tries to get an older version installed from another package (I forgot the package that is installing the old version).

Reply:
Removed https://github.com/Automattic/sensei/pull/5159/commits/6e95f28539c96167f4e7a18b75958bdefff47e29